content/docs/04.5-devx/05-app-profile/00-create-app-profile.md

```diff
@@ -14,118 +14,38 @@ import Tooltip from "shared/components/ui/Tooltip";
 
 # Create an App Profile
 
-You can create as many App Profiles as you need to fit your various types of workloads. Each App Profile may contain multiple services, also called tiers. Palette Dev Engine also supports the creation of multiple versions of an App Profile. To learn more visit the App versioning documentation [page](/devx/app-profile/versioning-app-profile).
+You can create as many App Profiles as needed to fit various types of workloads on your Palette Virtual Clusters. Each App Profile can contain multiple services, also called layers in the App Profile stack. In addition, you can create multiple versions of an App Profile. For more information, visit [App Profile Versioning](/devx/app-profile/versioning-app-profile).
```

Comment:
We should add after this paragraph a sentence that explains what will happen next. Such as "use the following steps to create an app profile" or something along those lines.